model/feature_flags.go

```diff
@@ -50,7 +50,7 @@ func (f *FeatureFlags) SetDefaults() {
 	f.CollapsedThreads = true
 	f.EnableRemoteClusterService = false
 	f.AppsEnabled = false
-	f.PluginIncidentManagement = "1.14.3"
+	f.PluginIncidentManagement = "1.16.0"
```

Comment:
Let's set this to "blank" so that split.io doesn't get involved at all:
```suggestion
	f.PluginIncidentManagement = ""
```